docs/tutorial/comments.md

```diff
@@ -2,20 +2,16 @@
 sidebar_position: 2
 ---
 # Comments
-
-1. Comments can be used to explain Unv code.
-2. Comments can be used to prevent execution when testing code.
+---
+Comments are used to explain code or to tell the compiler or runtime to ignore these lines.Thses lines are esentially invisible to the compiler or runtime.
```

Comment:
```suggestion
Comments are used to explain code or to tell the compiler or runtime to ignore these lines. These lines are essentially ignored by the compiler or runtime.
```
they are always visible for compiler. compiler removes them or changes them to another syntax of comments.